### PR TITLE
display links panel in detail view

### PR DESCRIPTION
**Pull request by CtrlSpice**
With minimal functionality (read-only) and an under construction banner.

Please note: I temporarily commented out the go:embed functionality to make the rest of my front-end tasks easier. If there's a better way to go about this please let me know.

Also, there's screenshots:
 
links
link-day

### CODE REVIEW

File: desktop-exporter/app/components/detail-view/links-panel.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <SpanField
+              fieldName="dropped attributes count"
+              fieldValue={link.droppedAttributesCount}
+              hidden={!link.droppedAttributesCount}
+            />
+          </AccordionPanel>
+        </AccordionItem>
+      </li>
+    );
+  });
```
Comment (jmorrell) [inline desktop-exporter/app/components/detail-view/links-panel.tsx:72]: Once a `map` gets this large, I recommend breaking it out as a separate component

```jsx
linkListItems = links.map((link) => <LinkItem link={link}/>)
```

File: desktop-exporter/server.go [hunk 2]
```diff
@@ -79,25 +77,26 @@ func traceIDHandler(store *TraceStore) func(http.ResponseWriter, *http.Request)
...
-            return
-	}
-	writer.Write(indexBytes)
+	http.ServeFile(writer, request, "./desktop-exporter/static/index.html")
+	// indexBytes, err := assets.ReadFile("static/index.html")
+	// if err != nil {
+	// 	writer.WriteHeader(http.StatusInternalServerError)
+    //         return
+	// }
+	// writer.Write(indexBytes)
```
Comment (jmorrell) [inline desktop-exporter/server.go:86]: You might do something like:

```go
if os.Getenv("SERVE_FROM_FS") == "true" {
    http.ServeFile(writer, request, "./desktop-exporter/static/index.html")
} else {
    indexBytes, err := assets.ReadFile("static/index.html")
    if err != nil {
        writer.WriteHeader(http.StatusInternalServerError)
        return
    }
    writer.Write(indexBytes)
}
```

so you can run during development like:

```
$ SERVE_FROM_FS=true go run ./...
```

**Review by jmorrell — APPROVED** [review]
DALL·E 2023-01-26 08 16 06 - a corgi with a warning light on its head, digital art

File: desktop-exporter/app/components/detail-view/links-panel.tsx (hunk 1 above)
Comment (CtrlSpice) [reply desktop-exporter/app/components/detail-view/links-panel.tsx:72]: Done here and for the same issue in the events tab

File: desktop-exporter/server.go (hunk 2 above)
Comment (CtrlSpice) [reply desktop-exporter/server.go:86]: Thank you!

File: desktop-exporter/app/components/detail-view/detail-view.tsx [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import React from "react";
 import { Flex, Tab, TabList, TabPanels, Tabs } from "@chakra-ui/react";
 
+
```
Comment (jmorrell) [inline desktop-exporter/app/components/detail-view/detail-view.tsx:4]: nit, extra line. Prettier should consolidate these I thought?

**Review by jmorrell — APPROVED** [review]
DALL·E 2023-01-16 14 15 02 - A corgi eating an enormous pile of tacos, digital art

File: desktop-exporter/app/components/detail-view/detail-view.tsx (hunk 3 above)
Comment (CtrlSpice) [reply desktop-exporter/app/components/detail-view/detail-view.tsx]: It treats lines between imports as intentional for some reason. Not sure why. I'll be more careful.

**Pull request merged**